envoy/config/filter/http/jwt_authn/v2/config.proto

```diff
@@ -0,0 +1,237 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwt_authn.v2;
+
+import "envoy/api/v2/core/base.proto";
+import "envoy/api/v2/core/http_uri.proto";
+import "envoy/api/v2/route/route.proto";
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+// This message specifies how a JSON Web Token (JWT) can be verified.
```

Comment:
nit: reflow the comment, column limit is 100 characters (throughout)

Reply:
Will do

Reply:
No, it is 100 per our clang-format config. https://github.com/envoyproxy/data-plane-api/blob/master/.clang-format#L12

Reply:
I see.  Will update it.